Helper/Config.php

```diff
@@ -163,42 +161,79 @@ public function getEnvironmentVariable(string $envVarName): ?string
      *
      * @return void
      */
-    public function setIsMerchantActive(bool $value): void
+    public function setIsMerchantActive(?int $storeId, bool $isActive): void
```

Comment:
Can you please update the docblock?

Reply:
Forgot to update some of those yep, good catch

Reply:
Thanks for updating! But please check the second parameter name.